tests/components/videoQualityPanel-test.js

```diff
@@ -268,65 +272,96 @@ describe('VideoQualityPanel', function() {
     mockSkinConfig.controlBar.qualitySelection = {
       'format': 'resolution bitrate'
     };
-    var DOM = TestUtils.renderIntoDocument(
+    var wrapper = Enzyme.mount(
       <VideoQualityPanel {...mockProps} />
     );
 
-    checkQualityTexts(DOM, bitrateLabels);
+    checkQualityTexts(wrapper, bitrateLabels);
 
-    checkAriaLabels(DOM, bitrateLabels);
+    checkAriaLabels(wrapper, bitrateLabels);
 
-    var components = TestUtils.scryRenderedDOMComponentsWithClass(DOM, 'oo-quality-screen-content-wide');
+    var components = wrapper.find('.oo-quality-screen-content-wide').hostNodes();
     expect(components.length).toBe(0);
   });
 
   describe('keyboard navigation', function() {
-    var qualityPanel, qualityButtons;
+    var qualityPanel, qualityButtons, wrapper;
 
     var getMockKeydownEvent = function(target, keyCode) {
-      return {
-        _type: 'keydown',
-        target: target,
-        keyCode: keyCode,
-        preventDefault: function() {}
-      };
+      return new CustomEvent('keydown', {
+        detail: {
+          _type: 'keydown',
+            target: target,
+          keyCode: keyCode,
+          preventDefault: function() {},
+          alex: 'alex'
+        }
+      });
     };
 
     beforeEach(function() {
-      var DOM = TestUtils.renderIntoDocument(
+      wrapper = Enzyme.mount(
         <VideoQualityPanel {...mockProps} />
       );
-      qualityPanel = TestUtils.findRenderedDOMComponentWithClass(DOM, 'oo-quality-panel');
-      qualityButtons = qualityPanel.querySelectorAll('[' + CONSTANTS.KEYBD_FOCUS_ID_ATTR + ']');
+      qualityPanel = wrapper.find('.oo-quality-panel').hostNodes();
+      qualityButtons = qualityPanel.getDOMNode().querySelectorAll('[' + CONSTANTS.KEYBD_FOCUS_ID_ATTR + ']');
     });
 
     afterEach(function() {
-      document.activeElement = null;
+      if (document.activeElement) {
+        document.activeElement.blur();
+      }
     });
 
+    //TODO: I couldn't find an easy way to simulate the keydown event with the proper event target,
+      //so I'm calling onKeyDown manually for the below tests for now
     it('should focus on previous menu item when pressing UP or LEFT arrow keys', function() {
       var activeIndex = qualityButtons.length - 1;
-      document.activeElement = qualityButtons[activeIndex];
-      qualityPanel.dispatchEvent(getMockKeydownEvent(document.activeElement, CONSTANTS.KEYCODES.UP_ARROW_KEY));
+      qualityButtons[activeIndex].focus();
+      wrapper.instance().onKeyDown({
```

Comment:
Same comment as above, would be better to keep the `getMockKeydownEvent` helper function?

Reply:
Will do